### PR TITLE
Update LinkingLibraries.md

### PR DESCRIPTION
**Pull request by hwitsel**
**motivation** Updated documentation on the subject of linking libraries since rnpm is now part of React Native core
**Test plan** Only documentation is changed

### CODE REVIEW

**Comment by facebook-github-bot-1** [comment]
By analyzing the blame information on this pull request, we identified @JoelMarcey and @nicklockwood to be potential reviewers.

**Comment by facebook-github-bot-1** [comment]
Thank you for your pull request and welcome to our community. We require contributors to sign our Contributor License Agreement, and we don't seem to have you on file. In order for us to review and merge your code, please sign up at https://code.facebook.com/cla - and if you have received this in error or have any questions, please drop us a line at cla@fb.com. Thanks!

**Comment by satya164** [comment]
Can you sign the CLA please?

File: docs/LinkingLibraries.md [hunk 1]
```diff
@@ -27,32 +27,25 @@ error as soon as you try to use the library._
 
 ### Automatic linking
 
-"[rnpm](http://github.com/rnpm/rnpm)" is a community project that allows linking of native dependencies automatically:
+The React Native CLI has the ability to link and unlink libraries to your project.
```
Comment (satya164) [inline docs/LinkingLibraries.md]: > The React Native CLI can link and unlink libraries with native dependencies automatically. To link a library, first install it under your project:

**Comment by hwitsel** [comment]
Sure! It should be signed now. 

Kind regards,
Hugo

**Comment by facebook-github-bot-5** [comment]
@hwitsel updated the pull request - [view changes](https://github.com/facebook/react-native/pull/9851/files/49452b1f8f500d996a9cf9a6933be49075e8ecd1..4033e3bbf8e4f9d5404c0d1bfd15cd286b064290)

**Comment by satya164** [comment]
@hwitsel No, you've not signed the CLA.

Also I've requested changes. Please update the PR.

**Comment by facebook-github-bot-9** [comment]
@hwitsel updated the pull request - [view changes](https://github.com/facebook/react-native/pull/9851/files/4033e3bbf8e4f9d5404c0d1bfd15cd286b064290..5a8f706c8f825650e18f6e21a20d5b9886f5f35e)

**Comment by hwitsel** [comment]
@satya164 I've sent an e-mail to cla@fb.com for the CLA and updated the PR. I only found one change request is that correct?

**Comment by JoelMarcey** [comment]
@hwitsel Could you not sign the CLA at https://code.facebook.com/cla ?? That's generally easier than the manual process of emailing to `cla@fb.com`.

**Comment by hwitsel** [comment]
@JoelMarcey I did. But according to @satya164 that failed, re-submitting the form gave the error that my e-mail address was already registered, therefore I've send an e-mail to cla@fb.com.

**Comment by facebook-github-bot-0** [comment]
@hwitsel updated the pull request - [view changes](https://github.com/facebook/react-native/pull/9851/files/5a8f706c8f825650e18f6e21a20d5b9886f5f35e..1261469bda3855b27b4361b0d9b38eabeaaede25)

**Comment by satya164** [comment]
cc @bestander any idea what happened with the CLA?

**Comment by bestander** [comment]
Will post in internal group

**Comment by bestander** [comment]
@JoelMarcey, is there anything you can do?

**Comment by JoelMarcey** [comment]
@hwitsel Is the email address that you signed the CLA with the same as the email address associated with your GitHub account?

**Comment by JoelMarcey** [comment]
It looks like @hwitsel has some CLA on file, but it is not associated with any GitHub account right now. Trying to figure out the best way to remedy that.

**Comment by JoelMarcey** [comment]
@hwitsel Ok, after some internal discussions, we need to try one of two things:

1. ensure the email address you signed the CLA with is associated with your GitHub account and public so we can verify.

2. ensure the email address you signed the CLA with is associated with your GitHub account and make a commit with it (this can be to a throwaway repo on your account) such that GitHub associates it with your GitHub account.

cc: @sdwilsh

**Comment by hwitsel** [comment]
@JoelMarcey First I feel really bad the CLA registration is not working, it feels like I've stirred up a lot of trouble for a small documentation update.

I've done the actions you proposed:
1. I've made my e-mail address public in my github account.
2. I've created a repo and made a commit (https://github.com/hwitsel/cla-test, I will discard it when the CLA is ok).

I hope this will fix the problem.

**Comment by JoelMarcey** [comment]
@hwitsel no worries. We can get this solved now so it is easier for future pull requests :)

I will see how we can get what you did with our CLA tools internally so that it shows that you have signed it.

**Comment by JoelMarcey** [comment]
@hwitsel Ok. The CLA issue seems to have been resolved now. Can you rebase this pull request and then we should be able to land it, assuming your changes are still valid. Thanks.

**Comment by hwitsel** [comment]
Michal Ordon made the same changes while I/we were struggling with the CLA. The good thing is that the documentation has improved.
I will close this PR, or is there any other cleanup procedure preferred?

**Comment by bestander** [comment]
just closing is fine :)

**Pull request closed without merging**